linode/manager
# chore: Update Storybook to latest to resolve `CVE-2023-42282`

## Description 📝

- Updating Storybook will resolve https://github.com/linode/manager/security/dependabot/93
- Storybook is the only thing in our repo that depends on the `ip` package and they fixed this in v7.6.17 https://github.com/storybookjs/storybook/pull/26086
- Does the same thing as https://github.com/linode/manager/pull/10210 but updates the package that depended on `ip` instead

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-02-21 at 10 46 01 AM](https://github.com/linode/manager/assets/115251059/4b50d226-928e-4b4a-9faf-c2cd949335df)  |  ![Screenshot 2024-02-21 at 10 44 39 AM](https://github.com/linode/manager/assets/115251059/603216bb-9d76-4ff4-be32-6496317bbcb6)  |

## How to test 🧪

- Check out this PR locally
- Run `yarn` to ensure dependencies are up to date
- Run `yarn storybook` 
- Verify storybook works as expected
- Verify the `build-storybook` Github Action passes ✅

## As an Author I have considered 🤔
